### PR TITLE
Reorganizing Semantics work in progress

### PR DESCRIPTION
**Pull request by royfielding**
Addresses #368

### CODE REVIEW

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -172,14 +172,15 @@
    HTTP has been the primary information transfer protocol for the World Wide
    Web since its introduction in 1990. It began as a trivial mechanism for
    low-latency requests, with a single method (GET) to request transfer of a
-   presumed hypertext document identified by a given pathname (HTTP/0.9).
+   presumed hypertext document identified by a given pathname. This original
+   protocol is now referred to as HTTP/0.9.
```
Comment (reschke) [inline draft-ietf-httpbis-semantics-latest.xml:176]: Maybe reference <https://www.w3.org/Protocols/HTTP/AsImplemented.html>?

Comment (royfielding) [reply draft-ietf-httpbis-semantics-latest.xml:177]: I am pretty sure that we tried that years ago and the RFC editor refused it.

**Comment by royfielding** [comment]
So, this is pretty close. The sections are mostly organized (except the internals of Message Composition) and I haven't changed any significant content. The next step would be to consolidate redundant sections, move some of the header field descriptive content up to the associated features, and fill in remaining gaps.

Minor uglies like the IANA section additions (after generated tables) could be improved as well.

File: draft-ietf-httpbis-semantics-latest.xml (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-httpbis-semantics-latest.xml:177]: I'd try again.  They took a less stable URL to a WHATWG spec recently.

**Comment by royfielding** [comment]
Done. Maybe medium-rare.

**Pull request merged**